netcdf_to_gltf_converter/netcdf/xbeach/xbeach_data.py

```diff
@@ -108,27 +119,34 @@ def fill_value(self) -> int:
         """
         return -1
     
-    def shift_coordinates(self, shift_x: float, shift_y: float) -> None:
+    def shift_coordinates(self, shift_x: float, shift_y: float, shift_z: float, variables: List[str]) -> None:
```

Comment:
maybe out of scope for this issue.
Should shift_coordinates be part of the dataset class, or should this dataset class be an @dataclass?

Reply:
Good idea, I will look into it.